### ISSUE TITLE
qucs crash when editing component properties

### ISSUE THREAD

**Issue by kraus1049** [Qucs/qucs#513]
### Reproducing way

```
1. Put "dc Voltage source"
2. Right click and choose "Edit Properties"
3. Press Enter key twice
```

qucs will fail into segmentation fault after step 3.

### qucs version

```
$ qucs --version
Qucs 0.0.19 (9d924ca)
```

### Environment

```
＄ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04 LTS"

$ arch
x86_64
```

**Comment by andresmmera** [comment]
Hello, 

I've tracked down the seg fault. It seems that this [line](https://github.com/Qucs/qucs/blob/master/qucs/qucs/components/componentdialog.cpp#L723) should be:

  if ( row < prop->rowCount()-1) {

row is an integer ranging from 0 to prop->rowCount()-1, so Qucs tries to access a non-existing element...

prop->setCurrentItem(prop->item(row+1,0));

row+1 > prop->rowCount()-1 => here's the seg fault

Should I open a PR with the patch?

**Comment by in3otd** [comment]
> Should I open a PR with the patch?

why shouldn't you? :grin:

**Comment by andresmmera** [comment]
just checking :-)

**Comment by in3otd** [comment]
Fixed in #514.

Thanks @kraus1049 for reporting the issue!
